crates/topos-certificate-spammer/src/lib.rs

```diff
@@ -201,7 +201,18 @@ pub async fn run(
 
     // Is list of nodes is specified in the command line use them otherwise use
     // config file provided nodes
-    let target_nodes = if let Some(nodes) = args.target_nodes {
+    let target_nodes = if args.kubernetes {
+        // Kubernetes mode
+        if let (Some(dns), Some(number)) = (args.dns, args.number) {
+            (1..=number)
```

Comment:
@foo0x29a Does the node will start from `1` or `0` ?

Reply:
Indexes will start from `0`!